happybase/connection.py

```diff
@@ -101,11 +109,14 @@ class Connection(object):
     :param str table_prefix_separator: Separator used for `table_prefix`
     :param str compat: Compatibility mode (optional)
     :param str transport: Thrift transport mode (optional)
+    :param bool use_kerberos: Connect to HBase via a secure connection (default: False)
+    :param str sasl_service: The name of the SASL service (default: hbase)
```

Comment:
please make the parenthesized part just "(optional)". defaults can be seen in the function spec below, which ar also included in the docs.